assets/app/scripts/filters/resources.js

```diff
@@ -24,7 +24,8 @@ angular.module('openshiftConsole')
       "encodedDeploymentConfig": ["openshift.io/encoded-deployment-config"],
       "deploymentVersion": ["openshift.io/deployment-config.latest-version"],
       "displayName": ["openshift.io/display-name"],
-      "description": ["openshift.io/description"]
+      "description": ["openshift.io/description"],
+      "buildNumber": ["openshift.io/build.number"]
```

Comment:
we are in the middle of a massive discussion about what the format of docker labels should be....  do we have a defined/agreed form for annotations?

@liggitt